lib/checks/navigation/p-as-heading-evaluate.js

```diff
@@ -79,6 +79,24 @@ function pAsHeadingEvaluate(node, options, virtualNode) {
   const nextStyle = nextSibling ? getStyleValues(nextSibling) : null;
   const prevStyle = prevSibling ? getStyleValues(prevSibling) : null;
 
+  if (node && nextSibling) {
+    const optionsHeadingLength = options.lengthThreshold.headingLength || [];
+    const optionsParagraphLength =
+      options.lengthThreshold.paragraphLength || [];
+
+    const headingLength = node.textContent.trim().length * optionsHeadingLength;
+    const paragraphLength =
+      nextSibling.textContent.trim().length * optionsParagraphLength;
+
+    if (headingLength > paragraphLength) {
+      return true;
+    } else if (headingLength < paragraphLength / 2) {
+      return false;
+    } else if (headingLength < paragraphLength) {
+      return undefined;
+    }
```

Comment:
Should use numbers instead of arrays. example (naming TBD)

```js
passLength: 1,
failLength: 0.5

headingLength = node.textContent.trim().length
paragraphLength = nextSibling.textContent.trim().length

if (headingLength > paragraphLength * passLength){
  return true
}
else if (headingLength < paragraphLength * failLength) {
  return false
}
```